dotnet3.5/SNS/SNSMessageExample/SNSMessageExample/SNSMessage.cs

```diff
@@ -0,0 +1,61 @@
+﻿// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX - License - Identifier: Apache - 2.0
+
+using Amazon;
+using Amazon.SimpleNotificationService;
+using Amazon.SimpleNotificationService.Model;
+using System;
+using System.Threading.Tasks;
+
+namespace SNSMessageExample
+{
+    class SNSMessage
+    {
+        private AmazonSimpleNotificationServiceClient snsClient;
+
+        /// <summary>
+        /// Constructs a new SNSMessage object initializing the Amazon Simple
+        /// the Notification Service (Amazon SNS) client using the supplied
```

Comment:
Amazon Simple the Notification Service (Amazon SNS) -> remove 'the'

Reply:
Thanks for catching that! Fixed.